packages/near-membrane-base/src/__tests__/environment.spec.ts

```diff
@@ -130,7 +130,7 @@ describe('VirtualEnvironment', () => {
     });
 
     describe('VirtualEnvironment.prototype.remap', () => {
-        it('Skips index-like properties', () => {
+        it('does not skip index-like properties', () => {
             // Ignoring "Property 'assertions' does not exist on type '{...}'."
             // @ts-ignore
             expect.assertions(1);
```

Comment:
Before we had no logic to actually skip index-like properties in the membrane. It relied on the fact that mutations where allowed to some objects that weren't live. My hunch is  that we in this PR we allow an object whose prototype's prototype is null to be considered as "plain" (using ObservableMemebrane's implementation) and it hits on something with that.